File: docs/code_examples/playwright/java/chromium_connect.java
import com.microsoft.playwright.*;

public class Main {
    public static void main(String[] args) {
        try (Playwright playwright = Playwright.create()) {
            Browser browser = playwright.chromium().connect(
                "wss://playwright.xclouds.dev/v1.58/?api_key=YOUR_API_KEY"
            );
            Page page = browser.newPage();
            page.navigate("https://example.com");
            browser.close();
        }
    }
}

File: docs/code_examples/playwright/python/chromium_connect.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.connect(
        ws_endpoint='wss://playwright.xclouds.dev/v1.58/?api_key=YOUR_API_KEY'
    )
    page = browser.new_page()
    page.goto('https://example.com')
    browser.close()
